### DISCUSSION TITLE
Uninstall homebrew formulaes related to Imagemagick

### DISCUSSION THREAD

**Discussion by lmBored** [d12frosted/homebrew-emacs-plus#589] [Q&A]
Hi everyone,

I recently installed Emacs 29 with native comp and ImageMagick flags on my system. However, Homebrew also installed a bunch of formulas related to ImageMagick that I believe I will never use. As a result, my brew list looks messy and cluttered.

I would like to know if there's a way to uninstall ImageMagick and all the formulas that were installed alongside it?

Any guidance on how to achieve this would be greatly appreciated.

Thank you in advance!

**Comment by ttran134** [comment]
No

**Reply by lmBored** [reply]
how tf u find this

**Reply by lmBored** [reply]
got solution in the discord anyways
